From 0876591da78dc024e6adc0a03b8abe89f2647f61 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:50 -0400
Subject: [redhat] kabi: Adding symbol vmalloc_32 (mm/nommu.c)

Message-id: <3218c2a9019649f19cdc10eb50172343403b7da1.1410510773.git.poros@redhat.com>
Patchwork-id: 96079
O-Subject: [PATCH RHEL7.1 230/233] kabi: Adding symbol vmalloc_32 (mm/nommu.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    This function is used to allocate virtual memory
    in DMP if kmalloc/__get_free_pages fails to provide one.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/vmalloc_32
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/vmalloc_32
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/vmalloc_32

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/vmalloc_32  b/redhat/kabi/kabi-module/kabi_ppc64/vmalloc_32 
new file mode 100644
index 0000000..f1426c0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/vmalloc_32  
@@ -0,0 +1,2 @@
+#1-
+0xa0b04675 vmalloc_32 vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/vmalloc_32  b/redhat/kabi/kabi-module/kabi_s390x/vmalloc_32 
new file mode 100644
index 0000000..f1426c0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/vmalloc_32  
@@ -0,0 +1,2 @@
+#1-
+0xa0b04675 vmalloc_32 vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/vmalloc_32  b/redhat/kabi/kabi-module/kabi_x86_64/vmalloc_32 
new file mode 100644
index 0000000..f1426c0
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/vmalloc_32  
@@ -0,0 +1,2 @@
+#1-
+0xa0b04675 vmalloc_32 vmlinux EXPORT_SYMBOL
-- 
1.7.1